pkg/rpc/breaker.go

```diff
@@ -89,10 +91,36 @@ func newBackOff(clock backoff.Clock) backoff.BackOff {
 	return b
 }
 
-func newBreaker(clock clock.Clock) *circuit.Breaker {
+func newBreaker(ctx context.Context, name string, clock clock.Clock) *circuit.Breaker {
```

Comment:
`name` looks unused, I think you want to plumb it into the logger.

Reply:
Done.